hazelcast-jet-spring/src/main/resources/hazelcast-jet-spring-0.6.xsd

```diff
@@ -0,0 +1,158 @@
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns="http://www.hazelcast.com/schema/jet-spring"
+           xmlns:tool="http://www.springframework.org/schema/tool"
+           xmlns:hz="http://www.hazelcast.com/schema/spring"
+           targetNamespace="http://www.hazelcast.com/schema/jet-spring"
+           elementFormDefault="qualified"
+           attributeFormDefault="unqualified">
+
+    <xs:import namespace="http://www.springframework.org/schema/tool"
+               schemaLocation="http://www.springframework.org/schema/tool/spring-tool.xsd"/>
+    <xs:import namespace="http://www.hazelcast.com/schema/spring"
+               schemaLocation="hazelcast-spring-3.10.xsd"/>
+
+    <xs:element name="instance">
+        <xs:annotation>
+            <xs:documentation>
+                Configure the hazelcast jet instance
+            </xs:documentation>
+            <xs:appinfo>
+                <tool:annotation>
+                    <tool:exports type="com.hazelcast.jet.JetInstance"/>
+                </tool:annotation>
+            </xs:appinfo>
+        </xs:annotation>
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="hz:hazelcast-bean">
+                    <xs:sequence>
+                        <xs:element ref="hz:config" minOccurs="0"/>
+                        <xs:element name="instance-config" type="instance-config" minOccurs="0"/>
+                        <xs:element name="default-edge-config" type="edge-config" minOccurs="0"/>
+                        <xs:element name="properties" type="hz:properties" minOccurs="0"/>
+                    </xs:sequence>
+                </xs:extension>
+            </xs:complexContent>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:element name="client">
+        <xs:annotation>
+            <xs:documentation>
+                Configure the Hazelcast Jet client
+            </xs:documentation>
+            <xs:appinfo>
+                <tool:annotation>
+                    <tool:exports type="com.hazelcast.jet.JetInstance"/>
+                </tool:annotation>
+            </xs:appinfo>
+        </xs:annotation>
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="hz:hazelcast-bean">
+                    <xs:choice minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="spring-aware" type="xs:string" minOccurs="0"/>
```

Comment:
is it not possible to import these somehow? it looks like copy of what's already in Hazelcast

Reply:
I've re-used <hz:config> since it was defined as a top level element on imdg side. But these are defined inside the <hz:client> tag so I cannot refer to them like I did for <hz:config>